prime-angular-frontend/src/app/core/resources/health-authority-resource.service.ts

```diff
@@ -107,6 +106,30 @@ export class HealthAuthorityResource {
       );
   }
 
+  public getHealthAuthorityVendorSiteIds(healthAuthorityId: number, vendorId: number): Observable<number[]> {
+    return this.apiResource.get<number[]>(`health-authorities/${healthAuthorityId}/vendors/${vendorId}/sites`)
+      .pipe(
+        map((response: ApiHttpResponse<number[]>) => response.result),
+        tap((healthAuthoritySiteIds: number[]) => this.logger.info('HEALTH_AUTHORITY_SITES', healthAuthoritySiteIds)),
+        catchError((error: any) => {
+          this.logger.error('[Core] HealthAuthorityResource::isHealthAuthorityVendorInUse error has occurred: ', error);
```

Comment:
wrong method in error "isHealthAuthorityVendorInUse"

Reply:
changed to match new method name